### DISCUSSION TITLE
YoloX or other 'Lite' Models

### DISCUSSION THREAD

**Discussion by Klinefelters** [ultralytics/ultralytics#4021] [Ideas]
I know that the library can run yolov5, v8, and nas right now. Are there plans on adding YoloX or yolo-fastest? Each one offers an extremely light detection model (<2 Million Params) which is useful for edge deployment.

**Comment by glenn-jocher** [comment]
@Klinefelters hello! We're glad to see your interest in efficient models for edge deployment. Currently, our focus is on improving and supporting the YOLOv8 ecosystem, which includes a variety of models tailored for different performance needs, including lightweight options suitable for edge devices.

While we don't have YoloX or Yolo-Fastest directly integrated, YOLOv8 does offer models with a smaller footprint that can be quite effective for edge deployment scenarios. We're continuously working on optimizing our models for speed and efficiency, so keep an eye on our updates for advancements in this area.

For the most up-to-date information on our supported models and their capabilities, please refer to our documentation. If you have specific requirements or suggestions, we're always open to community feedback to help guide our future developments. Thanks for your support! 🚀
